ncm-grub/src/main/perl/grub.pm

```diff
@@ -97,14 +160,19 @@ sub Configure {
       $fulldefaultkernelpath=$prefix.'/'.$kernelname.'-'.$kernelversion;
   }
 
-  my $cons = undef;
-  my $consolepath = "/hardware/console/serial";
-  my $serialcons  = undef;
-  my $newserial = "";
-  my $newterminal = "";
-  if ($config->elementExists($consolepath)) {
-    my %consnode = $config->getElement($consolepath)->getHash();
+  my $editor = CAF::FileEditor->new($GRUB_CONF,
+                                owner => "root",
+                                group => "root",
+                                mode => 0400,
+                                log => $self);
```

Comment:
this is potentially dangerous, since this file might contain plain text password. not sure how to deal with this. i'd say that when no password is removed/added/modified (ie password configuration is missing), you could leave it.